pages/dois/DoisHandler.php

```diff
@@ -41,9 +41,7 @@ protected function getAppStateComponents(\APP\core\Request $request, array $enab
                     $commonArgs,
                     [
                         'apiUrl' => $request->getDispatcher()->url($request, PKPApplication::ROUTE_API, $context->getPath(), 'submissions'),
-                        'getParams' => [
-                            'stageIds' => [WORKFLOW_STAGE_ID_EDITING, WORKFLOW_STAGE_ID_PRODUCTION],
```

Comment:
My concern with removing this it will include all submissions that have been submitted, accepted or pending. Is it possible to capture things that are also published if they're in different stages? Would something like `WORKFLOW_STAGE_ID_PUBLISHED` work? I'm not sure, but I don't think we want to include things that haven't been reviewed and "accepted" in some way here.

Reply:
Hi @ewhanson, yes, true... hmmm...
What do you think about introducing a query parameter for the submissions API and controller, e.g. 'casAssignDOI' that would then consider submissions in editing, production as well as those having a publication published?
WORKFLOW_STAGE_ID_PUBLISHED is never used I believe...
Hmm.... :thinking: